src/components/home/messageCard.tsx

```diff
@@ -0,0 +1,64 @@
+import { Card, CardActionArea, CardContent, CardMedia, Typography } from '@mui/material'
+import { styled } from '@mui/material/styles'
+import { ReactElement } from 'react'
+import avatar from '../../../public/images/home/avatar.svg'
+
+export default function MessageCard(): ReactElement {
+  const Container = styled('div')`
+    && {
+      padding: 20px;
+      text-align: center;
+    }
+  `
+  const Space = styled('div')`
+    && {
+      padding-bottom: 50px;
+    }
+  `
+  return (
+    <Container>
+      <Card sx={{ maxWidth: 378 }}>
+        <CardActionArea>
+          <CardMedia
+            component="img"
+            height="140"
+            image={avatar.src}
+            alt="random"
+            sx={{
```

Comment:
if use multiple props in sx should create style component. some props it should set default from config